gauntlet/packages/gauntlet-solana-contracts/src/lib/utils.ts

```diff
@@ -7,3 +11,34 @@ export const divideIntoChunks = (arr: Array<any> | Buffer, chunkSize: number): a
   }
   return chunks
 }
+
+export const parseContractErrors = async (sendTx: Promise<String>, idl: Idl): Promise<string> => {
+  let txhash
+  try {
+    txhash = await sendTx
+  } catch (err) {
+    // Translate IDL error
+    const idlErrors = parseIdlErrors(idl)
+    let translatedErr = ProgramError.parse(err, idlErrors)
+    if (translatedErr === null) {
+      throw err
+    }
+    throw translatedErr
+  }
+  return txhash
+}
+
+export const makeTx = async (rawTx: RawTransaction[]): Promise<Transaction> => {
+  const tx = rawTx.reduce(
```

Comment:
This shouldn't be an async.
Also you can return directly
```suggestion
  return rawTx.reduce(
```